tests/unit/controllers/admin/seeder-test.js

```diff
@@ -1,119 +1,113 @@
-import { module, test } from 'qunit';
-import { setupTest } from 'ember-qunit';
 import { next } from '@ember/runloop';
-import EmberObject from '@ember/object';
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
 import sinon from 'sinon';
 
 const { spy, stub } = sinon;
 
 module('Unit | Controller | admin/seeder', function(hooks) {
   setupTest(hooks);
-  hooks.beforeEach(function() {
-    const controller = {
-      _destroyAll: stub().resolves(),
-      _saveRandomLibrary: stub().resolves(),
-      _saveRandomAuthor: stub().resolves('newAuthor'),
-      _generateSomeBooks: stub().resolves('books'),
-      authors: 'authors',
-      books: 'books',
-      libraries: 'libraries',
-    };
-    this.controller = this.owner.factoryFor('controller:admin/seeder').create(controller);
-  });
+  test('generateLibraries action', async function(assert) {
+    assert.expect(4);
+
+    const controller = this.owner.lookup('controller:admin/seeder');
```

Comment:
for some reason the `beforeEach` hook doesn't work anymore for setting up the controller

Reply:
Strange indeed. I'll have to dig into this more after we get these fixes in.